### PR TITLE
RBN-82-Match Figma design more closely

### PR DESCRIPTION
**Pull request by maxencije**
### What did you implement?

Styling has been refactored so the final view look more like figma project.

### More info

- Jira: [RBN-82](https://productdock.atlassian.net/browse/RBN-82)

- Figma: [Design](https://www.figma.com/design/x4SgW80BE24s9gAFHM3gXS/RBC---Novi-Sad%2C-2024?node-id=2134-1466&node-type=frame&t=KOGU7IYE7q9erIOO-0)

### CODE REVIEW

**Review by ssukovic — COMMENTED** [review]
Decrease margins on small screens for Let's start exploring component

File: src/pages/Homepage/BookCatalogueSection/BookCatalogueSection.module.css [hunk 1]
```diff
@@ -156,6 +160,7 @@
   justify-content: center;
```
Comment (ssukovic) [inline src/pages/Homepage/BookCatalogueSection/BookCatalogueSection.module.css]:
```suggestion
```

Removing this aligns cards to the left, it looks a bit tidier than central alignment on smaller screens

File: src/pages/Homepage/BookCatalogueSection/BookCatalogueSection.module.css [hunk 2]
```diff
@@ -1,6 +1,9 @@
 .catalogueWrapper {
-  margin-inline: 320px;
   margin-block: 160px;
+
+  /* @media (max-width: 1425px) {
+    margin-block: 100px;
+  } */
 }
```
Comment (alestojanovic) [inline src/pages/Homepage/BookCatalogueSection/BookCatalogueSection.module.css:3]: You can remove dead code.

File: src/pages/Homepage/BookCatalogueSection/BookCatalogueSection.module.css [hunk 3]
```diff
@@ -103,7 +106,8 @@
 
 @media (max-width: 1425px) {
   .catalogueWrapper {
-    margin-inline: 100px;
+    /* margin: 0 30px; */
+    /* margin-top: 100px; */
```
Comment (alestojanovic) [inline src/pages/Homepage/BookCatalogueSection/BookCatalogueSection.module.css]: You can remove the whole class if this code is not used.

File: src/pages/Homepage/Homepage.module.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  @media (max-width: 1280px) {
+    margin: 0 30px;
```
Comment (alestojanovic) [inline src/pages/Homepage/Homepage.module.css]: Do you need this margin? I don't see any change in UI when I remove it?

**Review by ssukovic — APPROVED** [review]

**Review by Aleksandra-Romanic — APPROVED** [review]

**Pull request merged**